pkg/build/node/node.go

```diff
@@ -120,14 +124,24 @@ func NewBuildContext(options ...Option) (ctx *BuildContext, err error) {
 		ctx.kubeRoot = kubeRoot
 	}
 	// initialize bits
-	bits, err := kube.NewNamedBits(ctx.mode, ctx.kubeRoot)
+	bits, err := kube.NewNamedBits(ctx.mode, ctx.kubeRoot, ctx.arch)
 	if err != nil {
 		return nil, err
 	}
 	ctx.bits = bits
 	return ctx, nil
 }
 
+func supportedArch(arch string) bool {
```

Comment:
why is this change in this PR?
seems unrelated

Reply:
ok, I've read the description now, scratch this comment 🙃

Reply:
I went to test this package and refactored to eliminate the panic and simplify.